Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrTextEmitter.kt

```diff
@@ -15,6 +15,7 @@ import java.lang.IllegalStateException
  *
  * [omitEmptyLines] is useful for testing output (e.g. diff calculating).
  */
+@Suppress("PARAMETER_NAME_CHANGED_ON_OVERRIDE")
```

Comment:
Why not fix properly by rename parameter?

Reply:
the name of parameter doesn't corresponds the semantics of variable so that why I've suppressed this warning

Reply:
Scope of this suppress can be reduced to `printer` object.